src/app/lista-nominal/ListContainer.tsx

```
@@ -0,0 +1,367 @@
+                            sortField: sorting[0].field,
+                            sortOrder: sorting[0].sort,
+                        }],
+                        filters: value,
+                        ine: user.perfis.includes(9) ? user.equipe : undefined,
+                        pagination,
+                        search: search,
+                    });
+                    setResponse(res.data);
+                    setErrorMessage('');
+                } catch (error) {
+                    setErrorMessage('Erro ao buscar dados, entre em contato com o suporte.');
```

Biome (1.9.4)

[error] This variable is unused.

Unused variables usually are result of incomplete refactoring, typos and other source of bugs.
Unsafe fix: If this is intentional, prepend error with an underscore.


(lint/correctness/noUnusedVariables)